plugins/VstBase/RemoteVstPlugin.cpp

```diff
@@ -1007,10 +1007,10 @@ bool RemoteVstPlugin::load( const std::string & _plugin_file )
 	}
 
 	const char id[5] = {
-		static_cast<char>(m_plugin->uniqueID >> 24),
-		static_cast<char>(m_plugin->uniqueID >> 16),
-		static_cast<char>(m_plugin->uniqueID >>  8),
-		static_cast<char>(m_plugin->uniqueID      ),
+		'0' + static_cast<char>(m_plugin->uniqueID >> 24),
+		'0' + static_cast<char>(m_plugin->uniqueID >> 16),
+		'0' + static_cast<char>(m_plugin->uniqueID >>  8),
+		'0' + static_cast<char>(m_plugin->uniqueID      ),
```

Comment:
Hmm... did you confirm this with the [steinberg headers](https://github.com/R-Tur/VST_SDK_2.4/blob/4610873fa433d263a5c4a68b5b8b59ffcf6ca6db/pluginterfaces/vst2.x/aeffect.h#L167-L168) before doing this?  I believe this proposed change is a sanitation effort, right?  I'd expect this to already be in the right format and already a 48+ value.

Reply:
I did not. This was purely an attempt to produce the same result as the original `sprintf()` without actually using `sprintf()`. I'll take a closer look.

Reply:
Sorry, when I said "propose change is a sanitization effort", I was strictly referring to the latest `'0' + `, which I think is making an assumption that these values aren't already within character range, no?

Reply:
Ok, [I went ahead and tested this](https://gist.github.com/tresf/383706cc5985ec4f43130873ca157d0d). The `'0'` must be removed and this should be just fine.

This test was done with `GTG DPC3.dll`:

```
Starting using Qt-native embedding.
RemotePlugin::DebugMessage: ######## BEFORE id: dpc3
RemotePlugin::DebugMessage: ######## AFTER id: ���c
unique ID: ���c
```